services/Authentication/src/Authentication.DataAccessLayer/Configurations/UserConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+using Authentication.DataAccessLayer.Entities.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Authentication.DataAccessLayer.Configurations;
+
+public class UserConfiguration : IEntityTypeConfiguration<User>
+{
+    public void Configure(EntityTypeBuilder<User> builder)
+    {
```

Comment:
I don't see where you are implementing this configuration. You need to implement via the ApplyConfigurationsFromAssembly(Assembly.GetExecutingAssembly()) method in DbContext